src/screens/Notifications/ConfirmEmailScreen.tsx

```diff
@@ -224,6 +226,9 @@ class ConfirmEmailScreen extends Component<Props, State> {
   });
 
   onError = () => {
+    if (this.props.notificationError) {
+      Alert.alert(this.props.notificationError);
```

Comment:
no alerts on designs, the error should be displayed under the code

Reply:
I will talk with the design team about that - this error shouldn't appear at all. But, again, for this backend version, it is rather for the tester to know what is going on.